hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -153,6 +171,7 @@ set_output_delay ${SPI_DEV_OUT_DEL} [get_ports SPI_DEV_D3]   -clock SPI_DEV_CLK
 #####################
 # SPI HOST clock   #
 #####################
+# In Bronze the SPI host desing is a duplication of DEV design. For now, over-constraining with 62.5MHz
```

Comment:
I wanted to talk to you guys a bit about this. 
Some have proposed we design a spi host that can run even faster (~100MHz) and rely on the design being able to tweak the sampling points. 

What do you guys think about this approach? Would it be a PD hardship? My first thought is that it should not technically matter, but we will have to constrain probably the max delay.

Reply:
SPI host timing is hard to close even at lower speeds (50-60MHz). We will need to discuss before doing so at 100MHz

Reply:
i'll add this as an item to our discussion.  This is not final, i just wanted to bring it up and get some feedback from you guys.